src/languages/en.js

```diff
@@ -338,6 +338,7 @@ export default {
     notFound: {
         chatYouLookingForCannotBeFound: 'The chat you are looking for cannot be found.',
         getMeOutOfHere: 'Get me out of here',
+        iouReportNotFound: 'Payment details you are looking for cannot be found.',
```

Comment:
Also the change i suggested here earlier doesn't seem to be applied, Basically, i suggest that it begins with,
```
The
```
so the complete text looks like,
```
The payment details you are looking for cannot be found.
```